compose/ui/ui-test-junit4/src/skikoTest/kotlin/androidx/compose/ui/test/ComposeUiSkikoTestTest.kt

```diff
@@ -299,7 +299,7 @@ class ComposeUiSkikoTestTest {
             assertThat(changes[0].type).isEqualTo(PointerType.Touch)
         }
         events[1].apply {
-            assertThat(type).isEqualTo(Enter)
+            assertThat(type).isEqualTo(Move)
```

Comment:
Hm, that's a bit weird that we don't receive enter/exit events from touch. So if I touched and then moved finger inside component, it should be entered.
But if it is by design then -- ok

Reply:
Yes, it is by design, Android behaves the same with touch.